<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>satpassdialog</class>
 <widget class="QDialog" name="satpassdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Satellite pass prediction</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>31</y>
     <width>681</width>
     <height>228</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="displayFormat">
         <string>dddd, d MMMM yyyy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="timecheckBox">
        <property name="text">
         <string>Include current time</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="activeSatBtn">
        <property name="text">
         <string>Show active satellites</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save passlist...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="satListWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string/>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTableWidget" name="tableWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string/>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>120</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>19</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>19</number>
      </attribute>
      <column>
       <property name="text">
        <string>Spacecraft</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Downlink [MHz]</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>AOS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Max elev.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Direction</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Duration</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Azimuth</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Latitude</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Longitude</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Range [km]</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Orbit</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
